modules/ROOT/pages/implementing-apikit-4-for-soap.adoc

```diff
@@ -0,0 +1,12 @@
+= Implementing a SOAP Interface
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Create an API by using Anypoint Studio and APIkit for SOAP to consume a remote WSDL API. Download this link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl example] to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.
```

Comment:
I would change this paragraph to something simpler like this and get rid of the instructions to download the example API which is already mentioned in "Before You Begin Using APIkit for SOAP" section.
```suggestion
Implement a WSDL API by using Anypoint Studio and APIkit for SOAP to consume a remote SOAP service. 
```

Reply:
OK! Thanks!